yarn.lock

```diff
@@ -28411,6 +28500,36 @@ webpack@^4.33.0, webpack@^4.38.0:
     watchpack "^1.7.4"
     webpack-sources "^1.4.1"
 
+webpack@^5.56.0:
```

Comment:
Why is an older version of webpack being added?

Reply:
Copied over `yarn.lock` file from `main` again and updated it now. I don't see this older webpack being added anymore.